### DISCUSSION TITLE
Question about Windows server 2022 - .NET Core Tools - Microsoft.NetCore.App for .NET 6

### DISCUSSION THREAD

**Discussion by acheungedc** [actions/virtual-environments#10214] [General questions]
For windows server 2022 - .NET Core tools : Microsoft.NetCore.App : Is there a reason why version 6.0.32 is not installed on this hosted agent? This is the last version for .NET 6 which we need before we have capacity to upgrade to .NET 8 in the near future. (Version 6.0.31 doesn't work for us).

**Comment by kishorekumar-anchala** [comment]
Hi @acheungedc ,

1) Is there a reason why version 6.0.32  ? --> it will come up with new image rollout.(20240714.1)

**Reply by acheungedc** [reply]
Hi. The reason we need version 6.0.32 is because this version is a suggested fix by Mend for a vulnerability found.

**Reply by kishorekumar-anchala** [reply]
> Hi. The reason we need version 6.0.32 is because this version is a suggested fix by Mend for a vulnerability found.

Okay, The new version will be available with next release (20240714.1).

**Reply by acheungedc** [reply]
Thanks. I see that the next release is 20240714.1 (The date is July 14 in the name of the next release). But when will release be deployed? I check the software page this morning and there is no change yet

**Reply by kishorekumar-anchala** [reply]
It will be available by EOW

**Comment by kishorekumar-anchala** [comment]
Hi @acheungedc ,

  The new version has been released , Could you please check now with the new version.
  
  Relase : 20240714.1

**Comment by samus-aran** [comment]
We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.
